### DISCUSSION TITLE
Disable preset inherited from config:base

### DISCUSSION THREAD

**Discussion by delps1001** [renovatebot/renovate#8868] [Request Help] [answered]
I want nearly all of what's in config:base, but I specifically don't want the "group:monorepos". What is the best course of action here? To just copy what's currently in the config here: https://docs.renovatebot.com/presets-config/#configbase, but without the "group:monorepos"?

**Comment by rarkins** [comment, marked as answer]
Add this to config after your extends array:

"ignorePresets": ["group:monorepos"]


I'm surprised you want to do that though. Most of those groups need to be upgraded together.

**Reply by delps1001** [reply]
@rarkins the main reason is I'm not ready to update them (the build is failing), and there is no way to ignore (close) the PR since it is "immortal"... Is there a way to work around the immortal-ness of these grouped PRs?

**Reply by rarkins** [reply]
I'd recommend the dependencyDashboard feature and then configure approval for whichever ones you want so that PRs aren't created automatically.

**Reply by delps1001** [reply]
@rarkins is it possible to enable the dependency dashboard for only grouped updates like monorepos and the recommended group presets?

**Reply by viceice** [reply]
No, `dependencyDashboard` needs enabled globally, but `dependencyDashboardApproval` can be enabled via packageRules or globally.

**Reply by delps1001** [reply]
@viceice oh interesting, do you know how I could enable dependency dashboard approval for only grouped dependencies (I have group:recommended and group:monorepos enabled)

**Reply by rarkins** [reply]
This might work:

{
  group: {
    dependencyDashboard: true
  }
}

It will make *all* group PRs require approval including any you define

**Reply by delps1001** [reply]
will try, thanks!

**Reply by rarkins** [reply]
In such a case make sure to enable dependencyDashboard explicitly too
